app/packages/core/src/components/Actions/ActionsRow.tsx

```diff
@@ -162,7 +162,9 @@ const Tag = ({
   const [available, setAvailable] = useState(true);
   const labels = useRecoilValue(fos.selectedLabelIds);
   const samples = useRecoilValue(fos.selectedSamples);
-  const readOnly = useRecoilValue(fos.readOnly);
+  const canTag = useRecoilValue(fos.canTagSamples);
+  const isSnapshot = useRecoilValue(fos.readOnly);
```

Comment:
I would suggest keeping it `readOnly` instead of `isSnapshot` so it can be used for other readOnly use cases in future